Spigot-API-Patches/0028-EntityPathfindEvent.patch

```diff
@@ -1,4 +1,4 @@
-From 0f4f69d6a81d66c4906f1c05cac664bbdb129cf7 Mon Sep 17 00:00:00 2001
+From 4d3308328933e8e7f55359054b1775d5d7b543eb Mon Sep 17 00:00:00 2001
```

Comment:
Hmm, why are these patches modified?

Can you do `git checkout d59a313d Spigot-API-Patches/00*` to revert these back